template/{{app_name}}/app/assets/stylesheets/_forms.scss

```diff
@@ -0,0 +1,19 @@
+@use "uswds-core" as *;
+
+.usa-hint {
+  ul,
+  ol {
+    margin-bottom: units(2);
+  }
+}
+.usa-legend--large {
+  @include u-measure(5);
+  padding-top: units(2);
+  margin-top: 0;
+  margin-bottom: units(1);
```

Comment:
@LeslieJAnderson from [latest in PFML Starter Kit](https://github.com/navapbc/pfml-starter-kit-app/blob/a23df2a2c9cbb0b8fae8eb4c3799dd670c0baf74/paidleave/app/assets/stylesheets/_forms.scss), we only add the margin-bottom if not displaying an error I think.

Was there a reason to not do that here?